### PR TITLE
Modify rule S1163: LaYC format

### PR DESCRIPTION
**Pull request by cristian-ambrosini-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [x] logical errors and incorrect information
- [x] information gaps and missing content
- [x] text style and tone

### CODE REVIEW

**Comment by antonioaversa** [comment]
@cristian-ambrosini-sonarsource : labels to be applied, for all relevant languages

**Review by antonioaversa — CHANGES_REQUESTED** [review]

File: rules/S1163/rspecator.adoc [hunk 1]
```diff
@@ -1,16 +1,14 @@
-include::../rule.adoc[]
-
 ifdef::env-github,rspecator-view[]
 
 '''
 == Implementation Specification
 (visible only on this page)
 
-include::../message.adoc[]
+include::./message.adoc[]
```
[the remark this answers is not in the document]
Comment (cristian-ambrosini-sonarsource) [reply rules/S1163/rspecator.adoc:7]: I decided to embed directly the content in the rspecator

File: rules/S1163/rspecator.adoc [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 
 '''
 == Comments And Links
 (visible only on this page)
 
-include::../comments-and-links.adoc[]
-endif::env-github,rspecator-view[]
+include::./comments-and-links.adoc[]
```
[the remark this answers is not in the document]
Comment (cristian-ambrosini-sonarsource) [reply rules/S1163/rspecator.adoc:13]: I decided to embed directly the content in the rspecator

**Review by antonioaversa — APPROVED** [review]
LGTM!

**Pull request merged**